demo/components/victory-tooltip-demo.js

```diff
@@ -172,4 +175,94 @@ class App extends React.Component {
   }
 }
 
+class VictoryTooltipInPortal extends VictoryTooltip {
+  static contextTypes = {
+    portalRegister: React.PropTypes.func,
+    portalDeregister: React.PropTypes.func
+  }
+
+  render() {
+    this.element = super.render();
+    return null;
+  }
+
+  componentDidUpdate() {
+    this.context.portalRegister(this, this.element);
+  }
+
+  componentWillUnmount() {
+    this.context.portalDeregister(this);
+  }
+}
+
+class SvgPortal extends React.Component {
+  componentWillMount() {
+    this.map = new Map();
```

Comment:
@boygirl So thinking about this a little more I think this can be done without any use of Map or anything similar, by having each child "request" a unique key (string or integer) for itself in `componentWillMount` and then subsequently using that, I'll open a PR to victory-core with at least the Portal component showing this.